tests/specs/send/send-stx.spec.ts

```diff
@@ -4,225 +4,212 @@ import {
   TEST_BNS_RESOLVED_ADDRESS,
   TEST_TESTNET_ACCOUNT_2_STX_ADDRESS,
 } from '@tests/mocks/constants';
-import { SendPage } from '@tests/page-object-models/send.page';
 import { SharedComponentsSelectors } from '@tests/selectors/shared-component.selectors';
 import { getDisplayerAddress } from '@tests/utils';
 
 import { FormErrorMessages } from '@app/common/error-messages';
 
 import { test } from '../../fixtures/fixtures';
 
-test.describe('send stx', () => {
-  test.describe.serial('tests on testnet', () => {
-    const amount = '0.000001';
-    let sPage: SendPage;
-    test.beforeAll(async ({ extensionId, globalPage, onboardingPage, homePage, sendPage }) => {
-      await globalPage.setupAndUseApiCalls(extensionId);
-      await onboardingPage.signInWithTestAccount(extensionId);
-      await homePage.enableTestMode();
-      await homePage.sendButton.click();
-      await sendPage.selectStxAndGoToSendForm();
-
-      sPage = sendPage;
-    });
+const amount = '0.000001';
 
-    test('that send max button sets available balance minus fee', async () => {
-      await sPage.amountInput.fill('.0001');
-      await sPage.amountInput.clear();
-      await sPage.amountInput.blur();
-      await sPage.sendMaxButton.click();
-      await sPage.amountInput.blur();
-      test.expect(await sPage.amountInput.inputValue()).toBeTruthy();
-      await sPage.goBackSelectStx();
-    });
+test.describe('send stx: tests on testnet', () => {
```

Comment:
I refactored these tests to stop using `test.describe.serial` as they were pretty unreliable and the `send-btc` ones were slightly more reliable. 

We have some work to do to improve these tests and make them reliable. If I run them locally for example they fail if I don't pass the `--debug` flag to slow them down